prov/efa/src/rdm/efa_rdm_pke_cmd.c

```diff
@@ -673,6 +673,22 @@ void efa_rdm_pke_handle_rx_error(struct efa_rdm_pke *pkt_entry, int err, int pro
 	EFA_DBG(FI_LOG_CQ, "Packet receive error: %s (%d)\n",
 	        efa_strerror(prov_errno), prov_errno);
 
+	if (!pkt_entry->ope) {
+		char ep_addr_str[OFI_ADDRSTRLEN];
```

Comment:
Suggest adding a comment as reminder, e.g. the error might come from write with immediate data.

Reply:
it can be from both RECV and RECV_WITH_IMM. Don't think we need to call immediate data specially. But I will add comments to describe it

Reply:
Updated